tests/data_interface/test_data_parameters.py

```diff
@@ -0,0 +1,514 @@
+"""
+Explicitly test the DataParameters class and its methods.
+This module contains unit tests for the DataParameters class, which is part of the climakitae.core.data_interface module.
+These tests cover the initialization, default values, update methods, and retrieval of data parameters.
+"""
+
+from unittest.mock import MagicMock, Mock, patch
+
+import geopandas as gpd
+import pandas as pd
+import pytest
+from shapely.geometry import box
+
+from climakitae.core.data_interface import DataParameters
+
+
+def mock_scenario_to_experiment_id(scenario, reverse=False):
+    """Mock implementation of scenario_to_experiment_id with additional scenarios"""
+    scenario_dict = {
+        "Historical Reconstruction": "reanalysis",
+        "Historical Climate": "historical",
+        "SSP 2-4.5": "ssp245",
+        "SSP 5-8.5": "ssp585",
+        "SSP 3-7.0": "ssp370",
+        "SSP 1-2.6": "ssp126",  # Add the missing scenario
```

Comment:
Are there talks of adding this scenario into code base? If not, I think we should remove it. Perhaps it is in our data catalog though and I’m not aware? 

I think SSP 1-2.6 (the “sustainability” approach) is pretty unlikely and not commonly used, although maybe I am wrong.

Reply:
Ope, thank you. Removed.